usaspending_api/references/models.py

```diff
@@ -401,10 +474,362 @@ class LegalEntity(DataSourceTrackedModel):
     individual = models.TextField(blank=True, null=True)
 
     def save(self, *args, **kwargs):
+        LegalEntity.update_business_type_categories(self)
         super(LegalEntity, self).save(*args, **kwargs)
 
         LegalEntityOfficers.objects.get_or_create(legal_entity=self)
 
+    @staticmethod
+    def update_business_type_categories(le):
+        # Create a new list of categories
+        categories = []
```

Comment:
Just to make sure I understand correctly--we're adding this biz type array to support searching and displaying those, but we're also keeping the long list of individual business designations on the model in case folks need to reference them individually?